app/controllers/api/v0/api_controller.rb

```diff
@@ -46,8 +46,8 @@ def status_code_for(error)
     code || begin
       error_name = error.class.name
       ActionDispatch::ExceptionWrapper.status_code_for_exception(error_name)
-    rescue StandardError => _
-      500
+            rescue StandardError => _error
```

Comment:
Ohhhh, I see, it’s trying to align to the `begin` keyword, but didn’t indent the main clause of the block because `Layout/BlockAlignment` is set to `either`. I guess the *right* fix is probably to indent lines 47-48, otherwise we’d have to disable the `Layout/RescueEnsureAlignment` rule.